clickhouse/hatch.toml

```diff
@@ -0,0 +1,17 @@
+[env.collectors.datadog-checks]
+
+[envs.default.env-vars]
+CLICKHOUSE_REPOSITORY = "clickhouse/clickhouse-server"
+
+[[envs.default.matrix]]
+python = ["27", "38"]
+version = ["18", "19", "20", "21.8", "22.7"]
+
+[envs.default.overrides]
+matrix.version.env-vars = [
+    { key = "CLICKHOUSE_VERSION" },
+    { key = "CLICKHOUSE_REPOSITORY", value = "yandex/clickhouse-server", if = ["18", "19", "20", "21.8"] },
```

Comment:
I thought about that but did not do it because `clickhouse/clickhouse-server` is the new default value and we will use this value for the next versions, so basically this list is not going to grow and will probably one day disappear once those versions will not be supported anymore. I hope that makes sense